ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/DisplayScenesInTabletopAR.pro

```diff
@@ -80,15 +81,6 @@ android {
     DEPENDPATH += $$PWD
 
     ANDROID_ABIS = arm64-v8a
-
-    DISTFILES += \
-        android/AndroidManifest.xml \
-        android/build.gradle \
-        android/gradle/wrapper/gradle-wrapper.jar \
-        android/gradle/wrapper/gradle-wrapper.properties \
-        android/gradlew \
-        android/gradlew.bat \
-        android/res/values/libs.xml
```

Comment:
Do you want to remove all this? I thought it was required still

Reply:
@ldanzinger We do. The problem is that when we generate these templates at an earlier time it causes problems down the road. When I was testing my changes it was complaining about not being able to find some libraries that were specified in the manifest. This was similar to a problem I ran into with James previously and if I remember right it was referencing a deprecated library then and we didn't know why. The problem was solved in this case by just redoing the process specified in the AR toolkit. So I think this is more of a headache than it's worth since following the steps in the toolkit is really straight forward and it should avoid the problem I kept running into.

So in replacement of trying to do it for the end user, I just made reference to the toolkit instructions for deploying to android/iOS in the readme.

Should we continue to provide these upfront? or leave it up to the user?